components/Times.js

```diff
@@ -51,10 +51,10 @@ const styles = StyleSheet.create({
   },
   listItemTitle: {
     fontWeight: "bold",
-    
+
```

Comment:
I've seen quite a bit of this stuff going on. It looks like it's probably from an auto linter, which I don't know if that necessarily would have anything to do with some of the problems, but I wouldn't quite rule it out yet. Just something to keep in mind, and maybe look into later if these types of problems persist.